2/greeter.rb

```diff
@@ -1,2 +1,7 @@
-puts 'Hello, World!' do
+print 'Please enter your name:' do
+end
+
+name=gets.chomp
```

Comment:
How did you find out about this?

Reply:
https://ruby-doc.org/docs/ruby-doc-bundle/Tutorial/part_02/user_input.html

Needed to stop the exclamation mark from being printed on a different line!